### DISCUSSION TITLE
更改 hexo 文章路径

### DISCUSSION THREAD

**Discussion by fez7yoe** [walinejs/waline#1140] [Q&A] [answered]
想请教
例如将 hexo <code>_config.yml</code> 的 <code>permalink: year/:month/:day/:title/</code> 改成 <code>permalink: posts/:hash/</code>
之前在 <code>permalink: year/:month/:day/:title/</code> 状态下的评论就看不到了，变成空的
但改回 <code>permalink: year/:month/:day/:title/</code> 又看的到了

环境：数据库 LeanCloud、服务端 Vercel

**Comment by lizheming** [comment, marked as answer]
LeanCloud 里需要统一修改下评论对应的 path 字段

**Comment by Mister-Hope** [comment]
多说一句，permalink对应的就是永久链接的概念，也就是这个地址应该一经生成永不更改，在一般情况下，即使源项目变更不受到源项目文件结构、名称的影响。

Waline优先选择此项作为文章标识符是没有任何问题的。

你非要改这个选项就意味着你放弃了之前所有的永久链接，正如@lizheming所说请自行变更数据库里的path字段。
